_inc/client/components/jetpack-notices/state-notices.jsx

```diff
@@ -85,7 +85,9 @@ class JetpackStateNotices extends React.Component {
 				);
 				break;
 			case 'site_blacklisted':
-				message = __( "This site can't be connected to WordPress.com." );
+				message = __(
+					"This site can't be connected to WordPress.com because it violates our Terms of Service."
```

Comment:
It would be nice to link here.

Reply:
Good idea, added in https://github.com/Automattic/jetpack/pull/11183/commits/3a61e1b6ab05a5f4a3ca14b19f41ac78ebaee8df.